iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/checks/WhitespaceBracesCheck.java

```diff
@@ -35,8 +35,9 @@ public class WhitespaceBracesCheck implements IacCheck {
   private static final String MESSAGE_CLOSE_BRACKETS = "Add a whitespace before }} in the template directive.";
   private static final String GROUP_NAME = "brackets";
   // The [^\r\n\t\f\v -] in regex is like \S (non-space character) and not dash
-  private static final Pattern OPEN_BRACKETS = Pattern.compile("(?<" + GROUP_NAME + ">\\{\\{)[^\\r\\n\\t\\f\\v -]");
-  private static final Pattern CLOSE_BRACKETS = Pattern.compile("[^\\r\\n\\t\\f\\v -](?<" + GROUP_NAME + ">\\}\\})");
+  private static final String EXCLUDING_CHARS = "\\r\\n\\t\\f\\v -";
+  private static final Pattern OPEN_BRACKETS = Pattern.compile("(?<" + GROUP_NAME + ">\\{\\{)(?!/\\*)[^" + EXCLUDING_CHARS + "]");
```

Comment:
Maybe, extract the part `(?!/\\*)` into a variable `EXCLUDING_COMMENT_START`? Took me a moment to understand what's going on